master/internal/stream/trials.go

```diff
@@ -139,7 +139,8 @@ func TrialCollectStartupMsgs(
 		Table("trials").
 		Column("trials.id").
 		Join("JOIN experiments e ON trials.experiment_id = e.id").
-		Join("JOIN projects p ON e.project_id = p.id")
+		Join("JOIN projects p ON e.project_id = p.id").
+		OrderExpr("trials.id ASC")
```

Comment:
question: would we also want to apply the same change to `getTrialMsgsWithWorkspaceID()` in Trials.go?

Reply:
I don't think it really makes a difference since we're not dependent on the order for hydrating the messages later on (at least not based the trial id)

It might be worthwhile to think about if client's expect an order for the messages they receive, maybe based on seq.

Reply:
I don't think we absolutely need to, since the bug this is fixing appears in `ProcessKnown`